build.rs

```diff
@@ -1,3 +1,5 @@
+println!("cargo::rustc-check-cfg=cfg(nightly)");
```

Comment:
I'm surprised it works. I would put it into the `main()`.

Reply:
It 'works', but isn't necessary and will be removed.